tests/core/editor/initialize_hidden.js

```diff
@@ -5,26 +5,38 @@ CKEDITOR.replaceClass = 'ckeditor';
 
 bender.test( {
 	'test auto initilization': function() {
-		CKEDITOR.on( 'instanceReady', function( evt ) {
-			if ( evt.editor.name == 'editor1' ) {
-				resume( function() {
-					assert.areEqual( '<p>ed1</p>', bender.tools.compatHtml( evt.editor.getData() ) );
-				} );
-			}
-		} );
+		var editor = CKEDITOR.instances.editor1;
 
-		wait();
+		// For Firefox 60.0.1+ this code is executed after 'instanceReady' event so listener won't be called.
+		// For that case just assert it right away.
+		if ( editor.instanceReady ) {
```

Comment:
Using undocumented property might be a little bit risky. I think more appropriate will be using
`editor.status === 'ready'` instead.
https://docs.ckeditor.com/ckeditor4/latest/api/CKEDITOR_editor.html#property-status